Improve class names (#45)
TeamDev-IP/JxBrowser-QuickStart: README.md

- [Swing](Gradle/Swing) — how to embed a component into a Swing application to display web content 
- [JavaFX](Gradle/JavaFX) — how to embed a component into a JavaFX application to display web content
- [SWT](Gradle/SWT) — how to embed a component into an SWT application to display web content

> - [Console](Gradle/Console) — how to work with a web page without displaying any GUI

Reviewer: ```suggestion
- [Console](Gradle/Console) — how to work with JxBrowser without displaying GUI.
```
